ui/pages/_document.js

```diff
@@ -3,34 +3,6 @@ import PropTypes from "prop-types";
 import Document, { Head, Main, NextScript, Html } from "next/document";
 import flush from "styled-jsx/server";
 
-/**
- * setupGA setups up the google analtyics in meshery UI
- *
- * This function may not be invoked here (server side) as "window" object does
- * not exist on the server side. Hence this function is stringified and
- * forced to execute on the client side only.
- *
- */
-function setupGA() {
-  window.dataLayer = window.dataLayer || [];
-
-  function gtag() {
-    dataLayer.push(arguments);
-  }
-
-  fetch("/api/user/prefs", { credentials: 'include' })
-    .then((res) => res.json())
-    .then((res) => {
-      if (res?.anonymousUsageStats) {
-        gtag("js", new Date());
-        gtag("config", "G-8Q51RLT8TZ", {
-          page_path: window.location.pathname,
-        });
```

Comment:
Do we need this logic of firing the tag only when `res?.anonymousUsageStats` is true ?

Reply:
@Udit-takkar might have some insight.

Reply:
we can directly call (without checking res?.anonymousUsageStats)  gtag  like here https://developers.google.com/tag-platform/gtagjs/install to track all users data anonymously. 

maybe this was added to allow users to opt out of anonymous tracking by checking user preferences and only then track the user behaviour

Reply:
Thanks @Udit-takkar for the info :+1: Is there any other change you would suugest ?

Reply:
We track by default. Users can opt out under their User Preferences.

Is this PR complete or does it need changes @Udit-takkar?

Reply:
@leecalcote @ayushthe1  ah sorry didn't saw this comment before.

Like you said that "you track by default but let user opt out in user preference". i can see this functionality in the removed code but i don't see this in the new added code that we are fetching user preferences and checking anonymousUsageStats and only then  tracking events to Google Analytics to record the page load and configuration information.